doc/whats_new.rst

```diff
@@ -17,15 +17,18 @@ Enhancements
 ~~~~~~~~~~~~
 
 - Extended feature naming to all univariate functions. If no specific function is implemented for a given univariate function and that the output has the same shape as the input on the feature dimension, then a generic list of feature names is passed. Possibility to rename these feature by passing ``ch_names`` to :func:`mne_features.extract_features`. By `Paul ROUJANSKY`_ in `#60 <https://github.com/mne-tools/mne-features/pull/60>`_.
+- Extended feature naming to all bivariate functions, with the same option to pass ``ch_names``  to :func:`mne_features.extract_features` as for univariate functions. By `Hubert Banville`_ in `#52 <https://github.com/mne-tools/mne-features/pull/63>`_.
 
 Bug
 ~~~
 
 - Fixed the behavior of the `edge` parameter in :func:`mne_features.univariate.compute_spect_edge_freq`. Valid values for this parameter are now `None` or a list of float between `0` and `1` (percentages). By `Jean-Baptiste SCHIRATTI`_ in `#52 <https://github.com/mne-tools/mne-features/pull/52>`_.
+- Fixed channel name replacement with overlapping channel names (e.g., Cz and FCz) in :func:`mne_features.feature_extraction._apply_extractor`. By `Hubert Banville`_ in `#63 <https://github.com/mne-tools/mne-features/pull/63>`_.
```

Comment:
don't document changes in private functions. Say which public functions are affected and how